src/Simplex/Chat.hs

```diff
@@ -3409,7 +3411,7 @@ processAgentMsgRcvFile _corrId aFileId msg =
 
 processAgentMessageConn :: forall m. ChatMonad m => VersionRange -> User -> ACorrId -> ConnId -> ACommand 'Agent 'AEConn -> m ()
 processAgentMessageConn vr user@User {userId} corrId agentConnId agentMessage = do
-  entity <- withStore (\db -> getConnectionEntity db vr user $ AgentConnId agentConnId) >>= updateConnStatus
+  entity <- (withStore (\db -> getConnectionEntity db vr user $ AgentConnId agentConnId) >>= updateConnStatus) `catchChatError` \e -> throwError $ ChatErrorAgent (CRITICAL True $ show e) Nothing
```

Comment:
we only want to catch database Busy/Locked errors here, as we throw exception if entity is absent